src/Screen/Fields/Quill.php

```diff
@@ -71,4 +72,16 @@ class Quill extends Field
         'value',
         'height',
     ];
+
+    /**
+     * The given options will be add to the Toolbar.
+     *
+     * @return self
+     */
+    public function toolbar(array $options): self
+    {
+        $this->set('toolbar', json_encode($options));
```

Comment:
We can define the value using `docblock`. 
And call the method in the constructor so that the value is converted to JSON.

```php
* @method Quill toolbar(array $options)
// ...

    public function __construct()
    {
        $this->addBeforeRender(function () {
                $toolbar = $this->get('toolbar');
                $this->set('toolbar', json_encode($value));
            }
        });
    }

```
This will allow defaults to be specified in the array, and the JSON will only be converted on HTML output.
What do you think about this?

Reply:
ok, this is beyond my knowledge. But if you say is better, must be.